apps/judicial-system/web/src/types/index.ts

```diff
@@ -238,6 +239,7 @@ export interface TempCase
     | 'appealedByRole'
     | 'appealRulingDecision'
     | 'defendants'
+    | 'CaseCustodyRestrictions'
```

Comment:
Should this not be requestedCustodyRestrictions?

Reply:
Yeb! Thanks, fixed